torch/testing/_internal/common_utils.py

```diff
@@ -2177,7 +2177,7 @@ def run_test_function(self):
 def skipIfNoXNNPACK(fn):
```

Comment:
Please annotate these functions ussing `Callable[P,R]`, otherwise any function / method they decorate loses all typing info.

Reply:
There's a separate set of PRs that I'm working on to deal with untyped decorators. I've been landing them slowly because rather than just changing just the file you're looking at they cause action-at-a-distance (fixing an untyped decorator can cause user code to suddenly get type errors so they often have to be landed using the more complex internal diff process).